dietpi/dietpi-software

```diff
@@ -721,6 +722,40 @@ INDEX_WEBSERVER_TARGET=${INDEX_WEBSERVER_TARGET:=-2}
 			fi
 
 		fi
+		
+		#OPENBAZAAR
+		if (( $OPENBAZAAR == 1 )); then
+		
+			INSTALL_DESCRIPTION='OpenBazaar'
+			Banner_Installing
+		    
+			#apt-get -y update && apt-get -y upgrade
+			apt-get -y install git
```

Comment:
Use ```AGI``` function to ```apt-get install```. AGI has error checks and some other stuff, see function ```AGI()``` for more details.